packages/MarkdownEditor-Tests.package/MarkdownDelimiterEmphasisTest.class/instance/testMatchesOpenerNot.st

```diff
@@ -0,0 +1,17 @@
+tests - matching
+testMatchesOpenerNot
```

Comment:
the assert statements in this test and in testOpenerAndCloserMatchesOpener are the same only one time its asserted and the other is denied, so the different setup should be in the test case name. This one here does not match, because both delimiters' lengths are not divisible by 3, right? And in the other test case they do match because they are both 3? Or do they not match in this test case, because their length is different?